__snapshots__/index.test.js.snap

```diff
@@ -12,6 +12,24 @@ Baz.propTypes = {
 };"
 `;
 
+exports[`babel-preset-zapier emotion testing emotion 1`] = `
```

Comment:
Where does this snapshot come from? I don't see any tests with this description 🤔